src/main/java/edu/wpi/grip/ui/ExceptionAlert.java

```diff
@@ -0,0 +1,191 @@
+                    + "as well as a short description of what you were doing when this dialog appeared.",
+            COPY_PASTE_LABEL_TEXT = "Please paste this into a new issue on the project's GitHub:";
+
+    private static final String systemOptions[] = {
+            "javafx.version",
+            "java.runtime.name",
+            "java.vm.version",
+            "java.vm.vendor",
+            "java.vm.name",
+            "java.runtime.version",
+            "java.awt.graphicsenv",
+            "javafx.runtime.version",
+            "os.name",
+            "os.version",
+            "os.arch",
+            "file.encoding",
+            "java.vm.info",
+            "java.version",
+            "sun.arch.data.model",
+            "sun.cpu.endian"
+    };
+
+    private final String exceptionMessage;
+    private final String systemInfoMessage;
+    private final Throwable initialCause;
+
+    private final ButtonType openGitHubIssuesBtnType = new ButtonType("Open GitHub Issues");
+    private final ButtonType closeBtnType = new ButtonType("Cancel", ButtonBar.ButtonData.CANCEL_CLOSE);
+
+    /**
+     * @param throwable The throwable exception to display this alert for.
+     * @param services  The host services, allows access to {@link HostServices#showDocument(String)} in order to display
+     *                  the issue website.
+     * @see <a href="http://code.makery.ch/blog/javafx-dialogs-official/">Inspiration</a>
+     */
+    public ExceptionAlert(final Parent root, final Throwable throwable, final HostServices services) {
+        super(AlertType.ERROR);
+        checkNotNull(throwable, "The Throwable can not be null");
+        checkNotNull(services, "HostServices can not be null");
+
+        this.exceptionMessage = generateExceptionMessage(throwable);
+        this.systemInfoMessage = generateSystemInfoMessage();
+        this.initialCause = generateInitialCause(throwable);
+
+        this.setTitle(initialCause.getClass().getSimpleName());
+        this.setHeaderText(initialCause.getMessage());
+
+        // Set stylesheet
+        this.getDialogPane().styleProperty().bind(root.styleProperty());
+        this.getDialogPane().getStylesheets().addAll(root.getStylesheets());
+
+        // Add two additional buttons
+        this.getButtonTypes().removeIf((buttonType) -> buttonType.equals(ButtonType.OK));
+        this.getButtonTypes().addAll(openGitHubIssuesBtnType, closeBtnType);
+
+
+        final GridPane dialogContent = new GridPane();
+        dialogContent.setMaxWidth(Double.MAX_VALUE);
+        dialogContent.setMaxHeight(Double.MAX_VALUE);
+
+
+        final Label issuePromptText = new Label(ISSUE_PROMPT_TEXT);
+        issuePromptText.setWrapText(true);
+
+        final Label issuePasteLabel = new Label(COPY_PASTE_LABEL_TEXT);
+        issuePasteLabel.setWrapText(true);
+
+        final TextArea issueText = new TextArea(issueText(""));
```

Comment:
See my comment above.  Having the github link allows us to have the anonymized stack trace in any issues, and still have the dialog show the actual stack trace without extra markdown stuff